src/configuration/keybindings.jsonc

```diff
@@ -9,7 +9,7 @@
     {
         "key": "space",
         "command": "vspacecode.space",
-        "when": "sideBarFocus && !inputFocus && !whichkeyActive"
+        "when": "!inputFocus && !whichkeyActive"
```

Comment:
I am a little bit hesitant to use this because this binding is specifically targeted to sidebar. The changes might have unintended trigger, like user won't able to use space key in custom view.

Another direction to this fix is to find a better `when` clause like line 6 to focus on particular view.